.github/workflows/cost-optimization-hpa-config-recommender.yml

```
@@ -28,7 +28,7 @@ jobs:
       - name: Set up Python 3.12.5
```

Remove the need for us to update the name (whenever we bump `python-version`):
```suggestion
      - name: Set up Python
```